internal/description/description.go

```diff
@@ -118,6 +118,7 @@ const (
 	DescribeOrganizationsAPIKey = "Get a specific Organization API Key."
 	UpdateOrganizationsAPIKey   = "Update an Organization API Key."
 	CreateOrganizationsAPIKey   = "Create an Organization API Key."
+	ListWhitelistAPIKeys        = "Get all whitelist entries that belong to {API-KEY-ID}."
```

Comment:
this is not too friendly, let's follow the same style of other like this
```suggestion
	ListWhitelistAPIKeys        = "Get all whitelist entries for a given API key."
```